k8s/prod/common/ccd/definition-store-api.yaml

```
@@ -29,6 +39,8 @@ spec:
         IDAM_USER_URL: "https://hmcts-access.service.gov.uk"
         ELASTIC_SEARCH_FAIL_ON_IMPORT: true
         DEFINITION_STORE_DB_MAX_POOL_SIZE: 32
+        DEFINITION_STORE_DB_NAME: ccd_definition_store
+        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-prod-db-v11-perftest.postgres.database.azure.com
```

```suggestion
        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-db-v11-prod.postgres.database.azure.com
```
Please check this value